docs/rest_api.md

```diff
@@ -302,8 +305,11 @@ Update an existing tracingstore
 
 The compound annotations are created as merged from the finished annotations associated with the Task/Project/TaskType. This merging is performed before this info request is answered and can be slow for large numbers of annotations. The merged annotations are then stored in a cache for a few minutes, but not on disk. If requested again within this time, the request will be answered more quickly, but newly finished annotations will not be included yet. This cache is shared between this info request and the download request.
 
-#### `v1` differences
- - The returned JSON contains the `isPublic` flag instead of the `visibility` field 
+#### Changes Introduced in `v3`
+ - Expects additional GET parameter `timestamp=[INT]` timestamp in milliseconds (time the request is sent)
+
+#### Changes Introduced in `v2`
+ - The returned JSON contains the `isPublic` flag instead of the `visibility` field
```

Comment:
I think, we have to switch the sentence if we want to keep this wording because the `visibility` field was introduced in v2

Reply:
Thanks, good catch! I changed it, could you have another look?